src/GuessHistory.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Flex, Spacer, Text } from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+
+export default function GuessHistory() {
+    const [guessHistory, setGuessHistory] = useState<JSX.Element[]>([])
```

Comment:
You don't normally have the ELements saved in state. You want to save the data that goes INTO the elements in state. Then you pass that down as props to components that turn the state into elements. I would save something like GuessHistory[ ] in state. Or even if you have a Guess type, it could be an array of those